chore: update tabmaster to v1.3.0

Tormak9970 opened the pull request:
<!-- Make sure to include your plugin name below! -->

# TabMaster

Updated logging and fixed bug with how the plugin behaved on non-english systems

## Checklist:

### Developer Checklist

- [x] I am the original author or an authorized maintainer of this plugin.
- [x] I have abided by the licenses of the libraries I am utilizing, including attaching license notices where appropriate.

### Plugin Checklist

- [x] I have verified that my plugin works properly on the Stable and Beta update channels of SteamOS.
- [x] I have verified my plugin is unique or alternatively provides more/alternative functionality to a similar plugin already on the store.

<!-- Ex: "**Yes/No**: ..." becomes "**Yes**: ..." -->

### Plugin Backend Checklist

- **No**: I am using a custom backend other than Python.
- **No**: I am using a tool or software from a 3rd party FOSS project that does not have it's dependencies [statically linked](https://en.wikipedia.org/wiki/Static_library).
- **No**: I am using a custom binary that has all of it's dependencies statically linked.

<!-- The following section is should be modified to fit the conditions for plugin testing found here: https://wiki.deckbrew.xyz/en/plugin-dev/review-and-testing -->

## Testing

<!-- Remove this box for SteamOS Stable/Beta testing if you use a custom or remote binary, for more info follow the URL in the comment above the testing section. -->
- [ ] Tested on SteamOS Stable/Beta Update Channel.

PartyWumpus commented:
Are these supposed to be here? https://github.com/Tormak9970/TabMaster/blob/df1c5f280d46815f29029225d27564002ccad0c8/src/lib/controllers/PluginController.ts#L26
![image](https://github.com/SteamDeckHomebrew/decky-plugin-database/assets/48649272/352aeaa7-d700-4d77-8300-edd4bd9fa6ba)


I think for these to be themed properly any non-text stuff has to come afterwards (put `window.console` after the text).
https://github.com/Tormak9970/TabMaster/blob/df1c5f280d46815f29029225d27564002ccad0c8/src/lib/controllers/LogController.ts#L8
![image](https://github.com/SteamDeckHomebrew/decky-plugin-database/assets/48649272/7ff8d5dc-fa74-45a7-8e78-4e879dc6db77)

Tormak9970 commented:
> 

ugh forgot to remove them. odd, it was working fine on edge. will look into it tmr

david50407 commented:
Although Settings page loaded successfully on 1.2.1, but Home setting in the settings page loaded failed on 1.2.1

Tormak9970 commented:
> Although Settings page loaded successfully on 1.2.1, but Home setting in the settings page loaded failed on 1.2.1

I believe we have this fixed on the dev branch. we have a few other issues we're fixing before we push them to this update pr

Tormak9970 commented:
hmm, @jessebofill looks like the updates for the settings patch didn't work quite right, it was included in v1.2.1

Tormak9970 commented:
This is ready for testing again. The changes since the last test can be found here: [release v1.3.0](https://github.com/Tormak9970/TabMaster/releases/tag/v1.3.0)

Tormak9970 commented:
> Although Settings page loaded successfully on 1.2.1, but Home setting in the settings page loaded failed on 1.2.1

this is fixed with the most recent version

PartyWumpus reviewed (approved):
Code looks fine and changes work well on my (Main branch) steam deck
